### ISSUE TITLE
keyboard users can see the tooltip describing no percent reduction for the status quo scenario

### ISSUE THREAD

**Issue by DevinCLane** [DevinCLane/long-covid-dalys#213]
<img width="445" height="183" alt="Image" src="https://github.com/user-attachments/assets/e5afddec-1a0f-4cfe-a5bf-63d6990e68f7" />
the keyboard users should be able to trigger this tooltip
